net/xinetd/files/xinetd.init

```diff
@@ -1,15 +1,119 @@
 #!/bin/sh /etc/rc.common
-# Copyright (C) 2006-2011 OpenWrt.org
+# Copyright (C) 2008 OpenWrt.org
 
 START=50
+STOP=10
 
-SERVICE_USE_PID=1
+USE_PROCD=1
 
-start() {
-	service_start /usr/sbin/xinetd -pidfile /var/run/xinetd.pid
+PROG="/usr/sbin/xinetd"
+
+PIDFILE="/var/run/xinetd.pid"
+
+CONF_FILE="/etc/config/xinetd"
```

Comment:
Is this variable needed?
I can only find it once in code https://github.com/openwrt/packages/pull/12389/files#diff-6f5884fed5ef0d2076baa0e53c31f1a9R94

Reply:
Oh yes. I think you are right.
This is obviously a relict from former implementation.
Maybe we can remove it.